packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -52,6 +54,13 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   const errorBarProps = usePropsFor(ErrorBar);
   const adapter = useAdapter();
   const deviceState = adapter.getState().devices;
+
+  let disableStartCallButton = !microphonePermissionGranted || deviceState.microphones?.length === 0;
+  /* @conditional-compile-remove(rooms) */
+  const permissions = _usePermissions();
+  /* @conditional-compile-remove(rooms) */
```

Comment:
this logic needs some time to understand for reader, would be nice to add some comments to explain permissions.microphoneButton will impact the logic of startButton